portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,26 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> favQuotes;
+
+  @Override
+  public void init() {
+    favQuotes = new ArrayList<>();
+    favQuotes.add(
+      " \"And little by little, she found the courage for it all.\" -- JH Hard");
+    favQuotes.add(
+      "\"Be loud about the things that are important to you.\" -- Karen Walrond");
+    favQuotes.add(
+      "\"Le monde est un livre dont chaque pas nous ouvre une page.\""
+      + " -- Alphonse de Lamartine"
+      + "\n *** Translation: The world is a book - with every step, we open a page. ***");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    //return the ArrayList as JSON string
+    String json = new Gson().toJson(favQuotes);
     response.setContentType("text/html;");
```

Comment:
The content type is no longer an html text :) Please update it to `application/json`
[JavaDoc](https://docs.oracle.com/javaee/6/api/javax/servlet/ServletResponse.html#setContentType(java.lang.String))
[Common MIME types](http://go/mdn/HTTP/Basics_of_HTTP/MIME_types/Common_types)

Reply:
Thanks!